plugins/woocommerce/includes/emails/class-wc-email-customer-pos-refunded-order.php

```diff
@@ -447,9 +449,9 @@ private function register_refund_email_triggers() {
 		 * @return array Modified array of total rows.
 		 */
 		public function order_item_totals( $total_rows, $order, $tax_display ) {
-			$cash_payment_change_due_amount           = $order->get_meta( '_cash_change_amount', true );
-			$formatted_cash_payment_change_due_amount = wc_price( $cash_payment_change_due_amount, array( 'currency' => $order->get_currency() ) );
-			if ( ! empty( $cash_payment_change_due_amount ) ) {
+			$cash_payment_change_due_amount = $order->get_meta( '_cash_change_amount', true );
```

Comment:
As above

Reply:
Removed in https://github.com/woocommerce/woocommerce/pull/58012/commits/ae9001a162e6f85ddbb599cdb37486761b45711d.